### PR TITLE
Fix upgrade-consistency and version-service test

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

File: e2e-tests/version-service/conf/version-service-exact-rs0.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: version-service-exact
 spec:
   image: perconalab/percona-server-mongodb-operator:master-mongod4.0
+  initImage: perconalab/percona-server-mongodb-operator:master
```
Comment (delgod) [inline e2e-tests/version-service/conf/version-service-exact-rs0.yml:7]: no way, this field calculated from an operator image and different for each feature branch

**Review by delgod — CHANGES_REQUESTED** [review]

File: e2e-tests/version-service/conf/version-service-exact-rs0.yml (hunk 1 above)
Comment (blez) [reply e2e-tests/version-service/conf/version-service-exact-rs0.yml:7]: Should be `perconalab/percona-server-mongodb-operator:${branch}` ?

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade | passed |
| self-healing | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| operator-self-healing | passed |
| security-context | passed |
| scheduled-backup | passed |
| arbiter | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | failed |

File: e2e-tests/version-service/conf/version-service-exact-rs0.yml (hunk 1 above)
Comment (blez) [reply e2e-tests/version-service/conf/version-service-exact-rs0.yml]: https://github.com/percona/percona-server-mongodb-operator/pull/452/commits/244e13a87dcb41a4e1e2ab4fd58e07913b98a181

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| init-deploy | passed |
| storage | passed |
| upgrade | passed |
| self-healing | passed |
| monitoring | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| security-context | passed |
| arbiter | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| smart-update | passed |
| version-service | passed |
| users | failed |

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**